target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -11,6 +11,11 @@ board=$(board_name)
 boardname="${board##*,}"
 
 case "$board" in
+comfast,cf-e370ac)
+	ucidef_set_led_wlan "wlan" "WLAN" "comfast:blue:wlan" "phy0tpt" "phy1tpt"
```

Comment:
please change your label to follow the "boardname:color:function" rule.

Reply:
OK.